### PR TITLE
Closes AgileVentures/MetPlus_tracker#183

### PR DESCRIPTION
**Pull request by sidhene**

### CODE REVIEW

**Comment by sidhene** [comment]
Reviewers: you can mostly disregard the changes to jobseekers controller and controller spec files. The only substantive change is on line 105 on the spec.

**Comment by patmbolger** [comment]
Looks OK to me.

Related to this, could you add to this PR a change in spec/factories/job_seeker_statuses.rb from:

```
FactoryGirl.define do
  factory :job_seeker_status do
    value "MyString"
    description "MyTextString"
  end
end
```
to:
```
FactoryGirl.define do
  factory :job_seeker_status do
    value "Employedlooking"
    description "Currently employed but looking for better opportunity"
  end
end
```

**Comment by sidhene** [comment]
Sure, np. Done. Pushed.

**Comment by joaopapereira** [comment]
Do we have any test to check when we try to save the model without the status?

**Comment by joaopapereira** [comment]
I understand this pr undo a lot of space and stuff but the changes are not related with this PR we should try to minimize these changes that are unrelated

**Comment by sidhene** [comment]
@joaopapereira - that test is line 18 of "spec/models/job_seeker_spec.rb" as described above

Since git tracks any and all changes, I'm not sure that combing through "spec/controllers/job_seekers_controller_spec.rb" to make sure every line of code is in the same position it was before will improve the situation. Most likely,  that will only introduce even more changes which will then require yet another review.

File: spec/models/job_seeker_spec.rb [hunk 1]
```diff
@@ -15,9 +15,10 @@
 
   	it {is_expected.to validate_presence_of(:year_of_birth)}
   	xit {is_expected.to validate_presence_of(:resume)}
+    it {is_expected.to validate_presence_of(:job_seeker_status_id)}
```
Comment (joaopapereira) [inline spec/models/job_seeker_spec.rb]: Is the indentation correct?

**Comment by joaopapereira** [comment]
@sidhene Do not understand your comment

**Comment by sidhene** [comment]
Meaning: "spec/controllers/job_seekers_controller_spec.rb" has a lot of pink and green lines that reflect code that was added and removed. So the net is no change, or minor hard-to-trace changes like spaces. Since they're not "real changes" (like adding, replacing, or deleting code) they are not human-readable. They only show up in Git. If I try to do as you ask, I can't guarantee that the net effect won't be only more of the same.

**Comment by joaopapereira** [comment]
I understand that but we should not be reformatting the code per commit and editor.

**Comment by sidhene** [comment]
I'm not. I started to write some tests. They were the wrong tests. I deleted them. So I've already removed the changes.

File: spec/models/job_seeker_spec.rb (hunk 1 above)
Comment (joaopapereira) [inline spec/models/job_seeker_spec.rb]: Is the indentation correct in this line?

**Comment by joaopapereira** [comment]
@sidhene can you please check my comment in job_Seeker_spec.RB ?

File: spec/models/job_seeker_spec.rb (hunk 1 above)
Comment (sidhene) [inline spec/models/job_seeker_spec.rb]: it's good

**Comment by joaopapereira** [comment]
Going to merge, thanks for checking

**Pull request merged**